config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -105,6 +105,47 @@ periodics:
     testgrid-tab-name: build-master-canary
     testgrid-alert-email: release-managers+alerts@kubernetes.io, release-team@kubernetes.io
 
+- name: ci-kubernetes-build-no-bootstrap
+  interval: 1h
+  cluster: k8s-infra-prow-build
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-releng/k8s-ci-builder:latest-cross1.15
+      command:
+      - /krel
+      args:
+      - ci-build
+      - --allow-dup
+      - --fast
+      - --bucket=k8s-release-dev
+      - --gcs-suffix=no-bootstrap
+      - --docker-registry=gcr.io/k8s-staging-ci-images
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      resources:
+        limits:
+          cpu: 7300m
+          memory: "34Gi"
+        requests:
+          cpu: 7300m
+          memory: "34Gi"
+  rerun_auth_config:
+    github_team_ids:
+      - 2241179 # release-managers
+  annotations:
+    testgrid-dashboards: sig-release-master-informing, sig-release-releng-informing
```

Comment:
nitpick: should this also be on sig-testing-canaries or do you think no since the changed functionality is solely owned by sig-release?

Reply:
@hasheddan -- explicitly left it off of their board since the job is owned by SIG Release.